src/core/connection.c

```diff
@@ -1915,6 +1915,11 @@ QuicConnStart(
 #ifdef QUIC_OWNING_PROCESS
     UdpConfig.OwningProcess = Configuration->OwningProcess;
 #endif
+#ifdef QUIC_USE_RAW_DATAPATH
+    if (MsQuicLib.ExecutionConfig->Flags & QUIC_EXECUTION_CONFIG_FLAG_QTIP) {
+        UdpConfig.Flags |= CXPLAT_SOCKET_QTIP;
+    }
+#endif
```

Comment:
If we're always using QTIP when this is set, the DAL already knows this flag, so how about we make it automatic at that layer, instead of having QUIC logic for it?

Reply:
MsQuicLib isn't accessible to the DAL.

Reply:
ExecutionConfig is passed to datapath initialize.

Reply:
Done